packages/edit-site/src/components/editor/global-styles-provider.js

```diff
@@ -44,16 +45,19 @@ export default ( { children, baseStyles, contexts } ) => {
 	const nextValue = useMemo(
 		() => ( {
 			contexts,
-			getProperty: ( context, path ) =>
-				get( userStyles?.[ context ]?.styles, path ),
-			setProperty: ( context, path, newValue ) => {
+			getStyleProperty: ( context, propertyName ) =>
```

Comment:
I think we probably also need the old setters, otherwise for example to configure presets using the UI how are we going to do it? We also need setters and getters in global styles that don't map to a CSS property.

Reply:
So far, I don't believe we need anything else. Note that with the old format you could only change properties within `styles` anyway.

My understanding is that we'll need to create new presets for users, but I'm not sure how that's going to work because we're going to need access to both theme presets and user presets (unlike the current styles or features where we merge both). Perhaps in that case we could have a `getPreset( global, color, user )`/`setPreset( global, color, newValue )`. I guess we can cross that bridge when we get to it?

Reply:
Yes true we could only change style paths anyway. So I guess the change is positive :)